README.md

```diff
@@ -6,7 +6,7 @@ Copyright (C) Genymobile
 
 This repository stores Genymotion Android release keys, and corresponding .mk that we include internally in our `device.mk`. Third party developpers can use the release keys to sign their own applications in order to grant them system privileges on Genymotion. 
 
-**Note:** only the Android 10 version of Genymotion Device Image (PaaS) starting from v13.1 rely on these keys. Our other Android versions rely on the `test-keys` from AOSP. This note will be updated when we migrate other versions to the keys stored in this repository.
+**Notes:** Please refer to our [Root Access documentation page](https://docs.genymotion.com/paas/10_Using_root_access/) for more details about versions where this feature is available. Our other Android versions rely on the `test-keys` from AOSP. This note will be updated when we migrate other versions to the keys stored in this repository.
```

Comment:
Are we sure that the listing of the versions which support release builds will be added to the https://docs.genymotion.com/paas/10_Using_root_access/ page? I don't know if Jue started working on it

Reply:
Speaking of which, I've added @c-quenin and @cmajed as reviewer of the doc PR.